app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
Giải thích code này chi nghe nào

Reply:
Dạ chị. Nếu id người dùng không tồn tại, hàm sẽ tự động kết thúc và tự động trả về nil, hàm sẽ lưu kết quả User.find_by trong session

Reply:
cái này là viết tắt của:
```suggestion
    @current_user = @current_user || User.find_by id: session[:user_id]
```
nghĩa là nếu @current_user khác nil thì sẽ lấy giá trị của @current_user, còn nó nil thì sẽ tìm user theo session[:user_id]